app/src/main/java/com/example/inventory/ui/home/HomeScreen.kt

```diff
@@ -136,9 +139,12 @@ private fun HomeBody(
 
 @Composable
 private fun InventoryList(
-    itemList: List<Item>, onItemClick: (Item) -> Unit, modifier: Modifier = Modifier
+    itemList: List<Item>,
+    onItemClick: (Item) -> Unit,
+    contentPadding: PaddingValues,
+    modifier: Modifier = Modifier
 ) {
-    LazyColumn(modifier = modifier) {
+    LazyColumn(contentPadding = contentPadding, modifier = modifier) {
```

Comment:
nit: contentPadding should be after the modifier to match the function signature for LazyColumn

Also consider adding line breaks.

Reply:
Skipping line breaks.